check_stability.py

```diff
@@ -404,17 +404,22 @@ def get_branch_point(user):
         # To do this we need all the commits in the local copy
         fetch_wpt(user, "--unshallow", "+refs/heads/*:refs/remotes/origin/*")
         not_heads = [item for item in git("rev-parse", "--not", "--all").split("\n")
-                     if not head in item]
+                     if item.strip() and not head in item]
         commits = git("rev-list", "HEAD", *not_heads).split("\n")
-        first_commit = commits[-1]
-        branch_point = git("rev-parse", first_commit + "^")
+        branch_point = None
+        if len(commits):
+            first_commit = commits[-1]
+            if first_commit:
```

Comment:
This is unnecessary because of the `item.strip()` condition in the preceding list comprehension. You could remove that and keep this condition, but I think it's generally better to define the value in a more normal form.